jib-gradle-plugin/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
   id 'net.researchgate.release'
   id 'com.gradle.plugin-publish'
   // for local build/test
```

Comment:
Can we say `// for local SNAPSHOT build/install/test`? Unrelated to this PR, yesterday I couldn't figure out what this does and removed it in one of my branches.

Reply:
you mean the `maven-publish` plugin? In other sub-projects we use it to generate the poms.